src/Core/Enums/OrganizationUserType.cs

```diff
@@ -6,5 +6,6 @@ public enum OrganizationUserType : byte
         Admin = 1,
         User = 2,
         Manager = 3,
+        Custom = 4
```

Comment:
Please end all enums with a trailing comma, so when we add new ones (like you're doing here) we don't have to also modify the line above it.